__tests__/unit/fact.test.js

```diff
@@ -438,6 +439,44 @@ describe("getFact()", () => {
   });
 });
 
+describe("getAllFacts(num)", () => {
+  test("return all types of facts for given number", function () {
+    let validNum = getAllFacts(24);
+
+    expect(validNum.number).toBe(24);
+    expect(validNum.year.length).toBeGreaterThan(1);
+    expect(validNum.date.length).toBeGreaterThan(1);
+    expect(validNum.math.length).toBeGreaterThan(1);
+    expect(validNum.trivia.length).toBeGreaterThan(1);
+  });
+
+  test("return all types with default messages for missing facts", function () {
+    let validNum = getAllFacts(1932);
+
+    expect(validNum.number).toBe(1932);
```

Comment:
This will be a problem if facts get added for 1932. We need to set up a test db before this test is merged. This should probably be done in a separate pr.